Jenkinsfile

```
@@ -1528,7 +1550,9 @@ pipeline {
                         allOf {
                             not { branch 'weekly-testing' }
                             not { environment name: 'CHANGE_TARGET', value: 'weekly-testing' }
-                            // expression { ! skip_stage('scan-centos-rpms') }
+                            expression {
+                                ! commitPragma(pragma: 'Skip-scan-centos-rpms').contains('true')
+                            }
```

We have `skip_stage()` for detecting skipped stages.
```suggestion
                            expression { ! skip_stage('scan-centos-rpms') }
```

---

Same as above.